src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,147 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService regServ;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        regServ = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setLogin("someLogin");
+        user.setPassword("SomePassword_1234!");
+        user.setAge(21);
+    }
+
+    @Test
+    void register_emptyUser_NotOk() {
+        try {
+            regServ.register(null);
+        } catch (RuntimeException e) {
+            return;
+        }
+        fail("RuntimeException should be thrown if user is empty");
```

Comment:
```suggestion
        Assertions.assertThrows(RuntimeException.class, () -> regServ.register(null));
```
Change in each method accordingly